README.md

```diff
@@ -173,6 +173,32 @@ For [standalone installations](https://nix-community.github.io/home-manager/inde
   A: We primarily support the `unstable` branch, but try our best to support the current stable release.
   You can check if your stable release is currently supported at [status.nixos.org](https://status.nixos.org/)
 
+- Q: **"How do I fix the error "during evaluation because the option 'allow-import-from-derivation' is disabled"**\
+  A: Some ports require extra steps to configure the theme resulting in an [IFD](https://wiki.nixos.org/wiki/Import_From_Derivation).
+  You can manually disable that port (when using `catppuccin.enable`).
```

Comment:
```suggestion
  You can manually disable those ports (when using `catppuccin.enable`).
```

Reply:
Changed in a later commit since I forgot to do it before mirroring to doc/src/faq.md